lib/panko/attribute.rb

```diff
@@ -4,9 +4,17 @@ module Panko
   class Attribute
     def self.create(name, alias_name: nil)
       alias_name = alias_name.to_s unless alias_name.nil?
+      alias_name = transform_key(name.to_s) if alias_name.nil? && Panko.configuration.key_type
```

Comment:
I'm not entirely sure what the expected behavior should be when an alias is provided in snake case, but the configured key type is in camel case. What do you think? My suggestion is to camelize the alias if it's provided, but I'd like to hear your thoughts.

Reply:
I see you point, I think, while transforming keys is general setting for all keys and aliases is specific setting, then aliases should be in priority, like `developer has general settings with transforming, but for specific key he can specify alias`

and if you agree with this - then no changes is required in PR

Reply:
Agree with you, so please:
* Add test for this case, to make sure we don't change the alias
* In the documentation (https://github.com/yosiat/panko_serializer/pull/160/files#r1785809763) - we should note that (if you won't write the doc, I'll mention it..)

Reply:
I added test for using configuration with alias